README.md

```diff
@@ -39,12 +39,30 @@ Requirements
 * C++17 compiler
 * CMake >= 3.0.0
 * Google Test >= 1.10 (downloaded automatically)
-* Roc Toolkit (should be pre-installed system-wide, see [instructions](https://roc-streaming.org/toolkit/docs/building/user_cookbook.html))
+* Roc Toolkit (downloaded automatically)
 
 Instructions
 ------------
 
-Build and run:
+### Simple build
+When using this method, CMake will automatically download and build dependencies (Roc Toolkit). Roc will be statically linked into the modules and there is no need to install it into the system.
+
+First install build tools:
+
+```
+sudo apt install -y \
+    gcc g++ \
+    make \
+    libtool intltool m4 autoconf automake \
+    meson libsndfile-dev \
+    cmake \
+    scons \
+    git \
+    wget \
+    python3
```

Comment:
Is libsndfile-dev really needed?

Reply:
Replaced this with apt list from github actions.